deepcell_label/blueprints.py

```diff
@@ -102,6 +103,22 @@ def semantic_instance_labels(project_id, feature):
     return project.labels.cell_info[feature]
 
 
+@bp.route('/api/colormap/<project_id>/<int:feature>')
+def colormap(project_id, feature):
+    project = Project.get(project_id)
+    if not project:
+        return jsonify({'error': f'project {project_id} not found'}), 404
+    max_label = project.get_max_label(feature)
+    colormap = matplotlib.pyplot.get_cmap('viridis', max_label)
+    colors = map(matplotlib.colors.rgb2hex, colormap.colors)
+    # Add no label and new label colors
+    colors = ['#000000'] + list(colors) + ['#FFFFFF']
```

Comment:
Prefer to insert values into the colors list than to concatenate. As written this is creating 4 lists when we only want 1.